scipy/stats/tests/test_distributions.py

```diff
@@ -3981,7 +3981,7 @@ def setup_method(self):
         np.random.seed(1234)
 
     # skip these b/c deprecated, or only loc and scale arguments
-    fitSkipNonFinite = ['expon', 'norm', 'uniform']
+    fitSkipNonFinite = ['expon', 'norm', 'uniform', 'gilbrat']
```

Comment:
Uhm, this list looks strange. No shape parameters OK, why is gilbrat here? Ah, that's also because it's left in the `distcont` list.

Reply:
I put this here due to the comment directly above:
```
# skip these b/c deprecated, or only loc and scale arguments
```